### PR TITLE
added Passport number validator

### PR DESCRIPTION
**Pull request by kishanmodi**

### CODE REVIEW

File: types.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const EmailAddress = require('./validators/email');
 const PAN_IN = require('./validators/PAN_IN');
 const AadhaarNumber = require('./validators/Aadhaar_IN');
 const DrivingLC_IN = require('./validators/DRIVING_LC_IN');
+const passport = require('./validators/passport.js');
```
Comment (SwapnilSoni1999) [inline types.js]: remove `.js`

File: types.js [hunk 2]
```diff
@@ -12,4 +13,5 @@ module.exports = {
 	PAN_IN,
 	AadhaarNumber,
 	DrivingLC_IN,
+	passport,
```
Comment (SwapnilSoni1999) [inline types.js]: start variable name with `Passport`

File: validators/passport.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
+        return {
+            success: false,
+            message: 'Passport number should be string!' 
+        }
+    }
+
+    let panRegex = /^(?!^0+$)[a-zA-Z0-9]{6,9}$/;
```
Comment (SwapnilSoni1999) [inline validators/passport.js]: regex is built only once, use `const`

File: validators/passport.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            success: false,
+            message: 'Invalid PAN Card number!, Recheck the number please!!' 
+        }
+    }
+
+    return {
+        success: true,
+        message: "You Have Entered Valid Passport Number!!"
+    }
+}, 'passport')
```
Comment (SwapnilSoni1999) [inline validators/passport.js]: rename as `Passport`

**Review by SwapnilSoni1999 — CHANGES_REQUESTED** [review]
Please perform required changes

**Comment by kishanmodi** [comment]
added all the changes that were required

**Review by SwapnilSoni1999 — APPROVED** [review]

**Pull request merged**